### ISSUE TITLE
Accessibility: Drag and Drop Inaccessible to Visually Impaired Users

### ISSUE THREAD

**Issue by nibl** [Circa75/dropplets#143]
The current design for posting is entirely visual. Drag and drop is not accessible to visually impaired users. My suggestion would be an uploader.

Secondly, an uploader (a form with input type "file") would also make posting from mobile devices possible. I tried the following uploader with dropplets. The project is mature, stable, and works on mobile:

http://blueimp.github.com/jQuery-File-Upload/

Uploading on mobile is very browser dependent. Under the older Android 2.3, UCBrowser works but requires some geek knowledge (bonus: supports direct camera uploads). Under Android 4.2 several work to varying degrees. Firefox works best, supporting the internal file system, Dropbox and even direct camera uploads. I don't have an iOS device, but it is reported to work under iOS 6.x.
